src/encoded/static/components/pathology_report.js

```diff
@@ -0,0 +1,195 @@
+
+class PathologyReport extends React.Component {
+    constructor(props) {
+        super(props);
+    }
+    render() {
+        const context = this.props.context;
+        console.log(context);
+
+        const itemClass = globals.itemClass(context, 'view-item');
+        // Set up breadcrumbs
+        const crumbs = [
+            { id: 'PathologyReport' },
+            { id: <i>{context.name}</i> },
+        ];
+        const crumbsReleased = (context.status === 'released');
+
+        return (
+            <div className={itemClass}>
+                <header className="row">
+                    <div className="col-sm-12">
+                        <Breadcrumbs root="/search/?type=PathologyReport" crumbs={crumbs} crumbsReleased={crumbsReleased} />
+                        <h2>{context.name}</h2>
+
+                    </div>
+
+                </header>
+                <Panel>
+                    <PanelHeading>
+                        <h4>{this.props.chartTitle}</h4>
+                    </PanelHeading>
+                    <PanelBody>
+                        <dl className="key-value">
+                            <div data-test="status">
+                                <dt>Status</dt>
+                                <dd><Status item={context} inline /></dd>
+                            </div>
+                            <div data-test="surgery">
+                                <dt>Surgery</dt>
+                                <dd><a href={context.surgery}>{context.surgery.split("/")[2]}</a></dd>
+                            </div>
+                            <div data-test="date">
+                                <dt>Date</dt>
+                                <dd>{context.date}</dd>
+                            </div>
+                            <div data-test="laterality">
+                                <dt>Laterality</dt>
+                                <dd>{context.laterality}</dd>
+                            </div>
+                            <div data-test="tumor_size">
+                                <dt>Tumor_size</dt>
+                                <dd>{context.tumor_size}{context.tumor_size_units}</dd>
+                            </div>
+
+                            <div data-test="focality">
+                                <dt>Focality</dt>
+                                <dd>{context.focality}</dd>
+                            </div>
+                            <div data-test="histology">
+                                <dt>Histology</dt>
+                                <dd>{context.histology}</dd>
+                            </div>
+                            <div data-test="sarcomatoid">
+                                <dt>Sarcomatoid</dt>
+                                <dd>{context.sarcomatoid}</dd>
+                            </div>
+                            <div data-test="sarcomatoid_percentage">
+                                <dt>Sarcomatoid_percentage</dt>
```

Comment:
Hide the field sarcomatoid_percentage if there is no value for this field

Reply:
Same fix.